core/src/appleMain/kotlin/Peripheral.kt

```diff
@@ -407,5 +438,5 @@ private fun NSError.toStatus(): State.Disconnected.Status = when (code) {
     else -> Unknown(code.toInt())
 }
 
-internal actual val Peripheral.identifier: String
+public actual val Peripheral.identifier: String
     get() = (this as ApplePeripheral).platformIdentifier
```

Comment:
```suggestion
public typealias Identifier = Uuid

public actual val Peripheral.identifier: Identifier
    get() = (this as ApplePeripheral).platformIdentifier
```

Also requires that `platformIdentifer` to read:

```
internal val platformIdentifier = uuidFrom(cbPeripheral.identifier.UUIDString)
```